controllers/login.js

```diff
@@ -0,0 +1,51 @@
+const loginService = require("../services/login")
+
+
+const loginUser = async (req, res) => {
+    const { email, password } = { ...req.body }
+
+    if (email === undefined) {
+        return res.status(401).json({
+            error: "Email not provided"
+        })
+    }
+
+    if (password === undefined) {
+        return res.status(401).json({
+            error: "Password not provided"
+        })
+    }
+
+    try {
+        const user = await loginService.getUserByEmail(email)
+
+        if (user === null) {
+            return res.status(404).json({
+                error: "Email not found"
```

Comment:
Can we found better message. I will suggest to create utils file with all error messages for example "Not found" and than you can reuse in other places.